tests/other/test_tsearch.c

```diff
@@ -0,0 +1,64 @@
+#define _GNU_SOURCE
+
+#include <search.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <time.h>
+#include <assert.h>
+
+static void *root = NULL;
```

Comment:
This can be local in `main` maybe?

Reply:
yes, done. I push --forced with the new changes, thank you for the review, and sorry for the mistakes